releasenotes/notes/add-registry-to-flare-bb2b806f374633e8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Windows: Add Datadog registry to flare..
```

Comment:
```suggestion
    Windows: Add Datadog registry to Flare.
```

Reply:
Thank you, on it

Reply:
Fix it, but just realized that the commit comment contains `flare` in low-case. Hopefully will be more vigilant next time.